<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
"http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html>

<head>
<meta http-equiv="content-type" content="text/html; charset=iso-8859-1"/>
<meta name="description" content="This is the homepage of Charles-Elie Rabier. He mainly describes his research in statistics"/>
<meta name="keywords" content="QTL detection, Statistics, Gaussian Process, Charles-Elie Rabier, genetics"/> 
<meta name="author" content="Charles-Elie Rabier"/> 
<link rel="stylesheet" type="text/css" href="default.css" media="screen"/>
<title>Charles-Elie Rabier</title>
</head>

<body>

<div class="container">

	<div class="navigation">

		<div class="title">
			<h1>Charles-Elie Rabier</h1>
			<h2>welcome to my home page</h2>
		</div>
		<a href="doc/selectedTalks.html">Selected Talks</a> 
		<a href="doc/articles.html">Articles/Softwares/Conferences</a>
		<a href="index.html">Welcome</a>
		<div class="clearer"><span></span></div>

	</div>
 
	<div class="holder_top"></div>

	<div class="holder">
  	 
          
		<h1>Bonjour a tous</h1>
		<p>
I am associate professor at University of Angers, in applied mathematics (CNU 26).
 I am teaching statistics and applied maths at the faculty of Science, to students with a major in biology or chemistry, and also to students pursuing the Master Data Science. 
<br>
 <br>
Briefly speaking, in research, I am interested in stochastic processes arising in statistical genetics 
 (e.g. Gaussian and Chi-square processes) and in evolutionary biology (e.g. birth and death process, coalescent process ...). I also study asymptotic statistics, mixture models, Bayesian statistics and high dimensional data analysis.
 <br>
 <br>
 The applications are related to gene mapping, genomic prediction, genomic selection, phylogenetic network inference, reconciliation between gene trees and species tree, pangenomic and whole genome duplication detection.
<br>
<br>
Currently, in Angers, I am working with 
<a href="https://raschel.perso.math.cnrs.fr"> Kilian Raschel </a> (LAREMA) on phylogenetic network, funded by the PanphyloNet project. I am also mentoring Naïa Perinelle, PhD student, with <a href="https://irhs.angers-nantes.hub.inrae.fr/recherche/bioinformatics-for-plant-defense-investigations/composition-de-l-equipe-bidefi/claudine-landes">  Claudine Landès </a> (IRHS) and 
<a href="https://blog.univ-angers.fr/panloup/"> Fabien Panloup </a>(LAREMA) on 
machine learning for polyploid genomes.  I am also involved in the project <a href="https://biodivoc.edu.umontpellier.fr/recherche/projets-consortium/projet-de-consortium-devocgen/">
"DevOCGen"</a> supported by "region Occitanie".
<br>
 
<img src="img/cov.jpg" width="300" height="250"><img src="img/moy.jpg" width="300" height="250">
<br>
<br>
<br>
<br>



<h1>My email address</h1> 
<center>ce.rabieratgmail.com &nbsp;&nbsp; charles-elie.rabieratumontpellier.fr &nbsp;&nbsp; charles-elie.rabieratuniv-angers.fr</center>
<br>
<br>
<!-- 
<h1>My cv</h1>
<p>You can find here my cv in <a href="download/cvupdated2015.pdf">french </a> or in <a href="download/cvrabier2.pdf">english</a></p>
--> 
<br>
<h1>Research interests</h1> 
<p>
Asymptotic Statistics / Mixture models / Statistical Genetics / Gaussian Processes / Chi-Square Processes / 
Computational Biology /  Bayesian statistics / Phylogenetic networks  / Random Trees / Reconciliation between Gene Trees and Species Trees / High-dimensional data analysis 
<br>
<a href="https://www.researchgate.net/profile/Charles_Elie_Rabier">
<center><img src="img/ResearchGate.png"  width="100" height="50"></a>&nbsp;&nbsp;
<a href="https://github.com/rabier">
<img src="img/GitHub.jpg"  width="100" height="50"></center></a>
<br>
<h1>Member</h1> 
<br><center>
  <a href="https://www.sfds.asso.fr/"><img src="img/sfds.png" width="150" height="150"></a> &nbsp;&nbsp;
</center>
<br>
<h1> My postdoc on Bayesian statistics in evolutionary biology / phylogenetic network inference (University of Montpellier) </h1>
<br><center><img src="img/snapGoodNamesJDD6Run2Chro10avec1089SNPs.jpg" width="450" height="350"></center>
<br>


<br>
<p>
I worked at <a href="https://imag.umontpellier.fr">IMAG</a> and at ISEM, funded by the <a href="https://muse.edu.umontpellier.fr/key-initiatives-muse/data-life-sciences/">
 Key Initiative Muse Data Science (KIM) </a> and by the Genome Harvest project. 
KIM supported  my project "Statistics for pangenomic data", whereas Genome Harvest supported my project
on methodological aspects of phylogenetic networks inference. The application field  (collaboration with CIRAD) was 
the evolutionary history of rice and also the genome mosaic structure of citrus, banana, cocoa, coffee ...
We also organized the workshop <a href="https://ml4lifesciences.sciencesconf.org/">
"Machine learning for life sciences"</a> in Montpellier. 
During my postdocs, I worked in collaboration with
<a href="https://sites.google.com/view/celinescornavacca/open-positions">
 Celine Scornavacca</a>, <a href="https://www.lirmm.fr/~rivals/authors/fabio-pardi/">Fabio Pardi</a>, 
<a href="http://www.lirmm.fr/~vberry"> Vincent Berry</a> (computer scientists), <a href="https://scholar.google.com/citations?user=Pqqu9esAAAAJ&hl=fr&oi=ao">
Gilles Didier </a> (applied mathematician) and 
<a href="https://scholar.google.com/citations?user=2UFgMCoAAAAJ&hl=fr"> 
Jean-Christophe Glaszmann </a> (biologist). 
  
<p> 
In phylogenetics, species tree inference has been studied extensively for many years, and 
the theory behind it is relatively well known. 
However, a species tree is unable to model complicated biological events such as horizontal gene transfer 
(e.g. procaryotes but also among eucaryotes), 
hybridization (plants and animals, e.g. yeast, 
 wheat, fish), introgression (e.g. rice, citrus, sea bass) and recombination. 


In contrast, phylogenetic networks, that differ from species tree because of reticulate edges, are able to capture
 all those phenomenoms. Phylogenetic networks can help providing answers to important questions raised
 by biologists. For instance, we can infer the domestication process of cultivated rice 
by comparing different scenarios associated to different networks. 
Then, it becomes possible for geneticists to reintroduce some wild diversity into cultivars.  
According to the International Rice Research Institute, improving rice varieties may help in 
reducing poverty, in dealing with climate change, in protecting the environment, in making rice healthier, and
 in increasing food security.

I am interested in elaborating a new Bayesian method for inferring phylogenetic network. This method relies on 
 a complicated mathematical model, handling Incomplete 
lineage sorting (ILS) (an important source of discordance between gene trees), a mutation model
 (a markov process evolving along gene tree branches) ...
Our goal is to be able to infer the rice domestication process, with the help of our new sophisticated method.
<!-- 
See the slides (corresponding to different talks) at the bottom of this <a href="doc/articles.html"> webpage </a>
 -->
See the webpages dedicated to our simulator <a href="doc/SimSnappNet.html"> SimSnappNet </a>
and to the new method <a href="doc/SnappNet.html"> SnappNet </a>.
</p>


<br>
<h1>My postdoc on High dimensional data analysis /  Genomic prediction and Genomic Selection  (INRA Toulouse)</h1>
<br><center><img src="doc/figureGenomicSelection/GraphEmpAcc.PNG" width="300" height="300"></center>
<br>

<p>I was a member of the Applied Mathematics and Computer Science Unit at INRA in Toulouse (MIAT). 

I worked with Brigitte Mangin (statistician), and <a href="https://www.math.univ-toulouse.fr/~grusea/">
Simona Grusea </a> (probabilist, INSA Toulouse),
on high dimensional statistics and in particular on Ridge regression and its prediction when the number of SNPs (i.e. regressors) is larger than the number of individuals. 

The application was genomic selection. We collaborated with a few geneticists interested in different species :
<a href="https://www.researchgate.net/profile/Philippe_Barre"> Philippe Barre </a> (Raygrass, Lusignan),  
Gilles Charmet (Wheat, Clermont Ferrand), 
<a href="https://www.researchgate.net/profile/Jacques_David4"> Jacques David </a> and 
<a href="https://www.researchgate.net/scientific-contributions/2015420273_Muriel_Tavaud-Pirra">
Muriel Tavaud </a> (Wheat, SupAgro Montpellier).
Genomic selection is focused on prediction of breeding values of selection candidates by means of high density of markers. 
It relies on the assumption that all the genes 
 tend to be in strong linkage disequilibrium (LD) (ie highly correlated) with at least one marker. 
Our main goal was to find out the factors responsible for a good prediction, in a high dimensional setting. The R code for the Plos One paper
is available <a href="download/PROGRAMSFORSIMULATEDDATA.tar.gz"> here </a>.
See my articles either applied, or theoretical (perfect LD and imperfect LD). 
Many questions remain opened for statisticians in Genomic Selection.
</p>



<h1>My postdoc on stochastic processes in evolutionary biology  / Phylogenetics <br>(university of Madison, Wisconsin, USA)</h1>
<br><center><img src="img/yeastGeneTree.jpg" width="350" height="250">
</center>
<br>
<p>I was a member of the stat/math phylogenetic group at the university of Madison. This interdisciplinary group is composed of biologists, statisticians and mathematicians : <a href="http://botany.wisc.edu/baumlab/people/david-baum/"> David Baum</a>, <a href="https://www.stat.wisc.edu/~ane/"> Cécile Ané</a>, <a href="http://www.stat.wisc.edu/~larget/">Bret Larget</a>, <a href="http://www.math.wisc.edu/~roch/">Sebastien Roch</a> and students. 
The main goal was to propose new statistical methods to reconstruct evolutionary trees. 
Since more and more data from the genome are available, it becomes more and more interesting to study evolution events responsible for the diversity of genomes.  
Most of the methods involved in phylogenetics are Bayesian, since we have to deal with a huge parameter space.
We analyzed thousand of gene families and try to infer the phylogenetic tree for each of these families.
Since this work requires a huge amount of computer resources, we work in collaboration with Bill Taylor in order to use the technology offered by 
HTCondor, the product of years of research by the Center for High Throughput Computing in the Department of Computer Sciences at UW Madison.
HTCondor is used by Dreamworks in order to make movies and was also used in the recent discovery of the Higgs boson, a new particle in physics.
We were particularly interested in evolutionary events called Whole Genome Duplications (WGD), i.e. an event which creates an organism with additional copies of the entire genome of a species.
WGD can be found in bananas, tomatoes, flowering plants ... <a href="doc/SPIMAPWGD.html">Download our softwares for reconstructing gene trees and detecting WGD !</a>
</p>





<h1>My PhD on stochastic processes in statistical genetics / QTL mapping (university of Toulouse)</h1>
<br><center><img src="img/palmtreelanza1.jpg" width="290" height="400">
</center>
<br>
<p>
I completed my PhD at the Laboratory of Statistics and Probability (LSP) of Paul Sabatier's
university, Toulouse.
My supervisors were <a href="http://www.math.univ-toulouse.fr/~azais/"> Jean-Marc Azais </a>(LSP) and Jean-Michel Elsen (INRA). The research area of my PhD in mathematical statistics was on devising statistical methods for Quantitative Trait Locus detection.
 I worked on selective genotyping experiments and also on the empirical process defined by the famous Interval mapping of Lander and Botstein.</p>
<p>First, QTL means Quantitative Trait Locus. It is any locus responsible for the variation of a quantitative trait. On this picture, you can
see a few plamtrees which come from Lanzarote (Canary Islands). We may wonder why the first palmtree is largely taller than the others. There is maybe a QTL responsible for this
particularity. In order to locate the QTL, people scan the genome of the palmtrees and do a statistical test at each position. 
That is the principle of interval mapping (Lander and Botstein 89). 
Since a lot of tests are performed
on the genome, we are confronting to multiplicity problem. The work of Benjamini and Hochberg (95) on the false discovery rate is a way to deal with this 
problems of multi testing. Another approach is to study the stochastic process defined by all the correlated tests performed
 along the genome. I study the asymptotic properties of the corresponding empirical process. The Matlab pictures on the top of this web page 
gives a summary on the asymptotic properties of the empirical process. On the left, is represented the correlation whereas on the right, is given the 
mean function of the empirical process, that is to say the signal which contains information about the QTL.
On the other hand, genotyping was very expensive in the past.
That's why Lebotwitz (1987) proposed to genotype only individuals with extreme phenotypes : it is selective genotyping.
I study the asymptotic properties of such a design and study also the empirical process under selective genotyping.
Note that selective genotyping is still very useful nowdays, even if thousands of genetic markers are now available. 
We can increase the signal by genotyping only extreme individuals. 
 <a href="doc/articles.html"> Download our softwares to compute the quantiles of the maximum of Gaussian processes and Chi Square processes!</a>
</p>
<br>
<br>
<br>

</div>





<div class="webdesign"><p><a href="http://www.sarkis-webdesign.com/" title="Webdesign" class="web_design">Webdesign</a> service by Sarkis. <a
href="http://freelancewebmarket.com/" title="Outsourcing" class="outsourcing">Outsourcing</a> by FreelanceWebmarket.</p></div>

 <div class="footer">&copy; 2007 <a href="index.html">Website.com</a>. Valid <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> &amp; <a
href="http://validator.w3.org/check?uri=referer">XHTML</a>. <a href="http://templates.arcsin.se">Website template</a> by <a
href="http://arcsin.se">Arcsin</a>

 </div>

</div>

</body>

</html>
