<html>

<head>
<meta http-equiv="content-type" content="text/html; charset=iso-8859-1"/>
<meta name="description" content="description"/>
<meta name="keywords" content="keywords"/> 
<meta name="author" content="author"/> 
<link rel="stylesheet" type="text/css" href="../default.css" media="screen"/>
<title>Light</title>



<style>
.button {
  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 10px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 12px;
  margin: 4px 2px;
  -webkit-transition-duration: 0.4s; /* Safari */
  transition-duration: 0.4s;
  cursor: pointer;
}

.button1 {
  background-color: white; 
  color: black; 
  border: 2px solid #4CAF50;
}

.button1:hover {
  background-color: #4CAF50;
  color: white;
}

.button2 {
  background-color: white; 
  color: black; 
  border: 2px solid #008CBA;
}

.button2:hover {
  background-color: #008CBA;
  color: white;
}

.button3 {
  background-color: white; 
  color: black; 
  border: 2px solid #f44336;
}

.button3:hover {
  background-color: #f44336;
  color: white;
}

.button4 {
  background-color: white;
  color: black;
  border: 2px solid #e7e7e7;
}

.button4:hover {background-color: #e7e7e7;}

.button5 {
  background-color: white;
  color: black;
  border: 2px solid #555555;
}

.button5:hover {
  background-color: #555555;
  color: white;
}
</style>

</head>

<body>

<SCRIPT>
function passWord() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/PaperEthacOptim.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>




<SCRIPT>
function passWordISAE() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/talkISAE.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>


<SCRIPT>
function passWordJFRB() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/TalkReseauBayesienBis.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>



<SCRIPT>
function passWordMAB() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/TalkMAB.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>



<SCRIPT>
function passWordR2D2() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/TalkNoirmoutiers.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>






<SCRIPT>
function passWordGHDay() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/talkCIRADShortOneJDD.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>


<SCRIPT>
function passWordGHLastMe() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/TalkReseauCIRADLastGH.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>


<SCRIPT>
function passWordGHLastVincent() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/Presentations_VB-juin18.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>




<SCRIPT>
function passWordImperfectLD() {
var testV = 1;
var pass1 = prompt('Veuillez entrer le mot de passe',' ');
while (testV < 3) {
if (!pass1) 
history.go(-1);
if (pass1.toLowerCase() == "diaconis") {
alert('Mot de passe correct !');
window.open('http://charles-elie.rabier.pagesperso-orange.fr/download/PapierImperfectLD.pdf');
break ;
} 
testV+=1;
var pass1 = 
prompt('Accès interdit - Mot de passe incorrect. Veuillez réessayer.','Mot de passe');
}
if (pass1.toLowerCase()!="diaconis" & testV ==3) 
history.go(-1);
return " ";
} 
</SCRIPT>










<div class="container">

	<div class="navigation">

		<div class="title">
			<h1>Charles-Elie Rabier</h1>
			<h2>welcome to my home page</h2>
		</div>

		
		<a href="articles.html">Articles/Softwares/Conferences</a>
		<a href="../index.html">Welcome</a>
		<div class="clearer"><span></span></div>

	</div>

	<div class="holder_top"></div>

	<div class="holder">

		<h1>Publications
<a href="https://www.library.caltech.edu/eds?query=rabier%20charles-elie&pagenumber=2&ff[]=SourceType:Academic%20Journals&ff[]=SourceType:Magazines&pagenumber=1"> (via Caltech Library) </a> </h1>
		<p>Sabbadin R., Spring D., Rabier C.E.: <a href="http://www.sciencedirect.com/science/article/pii/S0304380006003577">  Dynamic reserve site selection under contagion risk of deforestation </a> (Ecological Modelling, Vol
201, Issue 1, February 2007, p75-81)</p>
            <p><a href="http://www.tandfonline.com/doi/abs/10.1080/02331888.2012.760093?journalCode=gsta20"> Likelihood Ratio Test process for Quantitative Trait
Locus Detection </a>, with Azais J-M and Delmas C (Statistics, 2012, DOI:10.1080/02331888.2012.760093)
</p>
<p style="color:#FF0000";>Data in Statistics : Rice data from Huang et al. (Molecular breeding, 1997)
</p>
<p> Rabier C-E, Genz A : <a href="http://link.springer.com/article/10.1007%2Fs11009-013-9331-1">The supremum of Chi-Square processes</a>, (Methodology and Computing in Applied Probability, 2013, DOI 10.1007/s11009-013-9331-1)</p>
<p>Rabier C-E : <a href="http://link.springer.com/article/10.1007/s11749-013-0349-z"> On Quantitative Trait Locus mapping with an interference phenomenon</a>, (TEST, Vol 23, 2014, p311-329)</p>
<p>Rabier C-E : <a href="http://www.sciencedirect.com/science/article/pii/S0378375813002966"> On statistical inference for selective genotyping</a>, (Journal of Statistical Planning and Inference, Vol 147, April 2014, p24–52)</p>
<p>Rabier C-E : <a href="http://www.sciencedirect.com/science/article/pii/S0378375814000974"> On empirical processes for quantitative trait
locus mapping under the presence of a selective genotyping and an interference
phenomenon</a>, (Journal of Statistical Planning and Inference, Vol 153, October 2014, p42-55)</p>
<p>Rabier C-E, Ta T., Ane C. : <a href="http://mbe.oxfordjournals.org/content/early/2014/01/13/molbev.mst263.abstract?keytype=ref&ijkey=Ls8mHjKto38Tzur">  Detecting and locating Whole Genome Duplications on a phylogeny: a probabilistic approach </a> (Molecular Biology and Evolution, 2014, doi:10.1093/molbev/mst263)
<p style="color:#FF0000";>Data in MBE: yeast data, same gene families as in Butler et al. (Nature 2009), that is to say 9209 gene families, downloaded from <a href="http://compbio.mit.edu/candida"> http://compbio.mit.edu/candida </a> </p>

<p>Rabier C-E : <a href="http://afst.cedram.org/afst-bin/fitem?id=AFST_2014_6_23_4_755_0">  An asymptotic test for Quantitative Trait Locus detection in presence of missing genotypes  </a> (Annales de la Faculté des Sciences de Toulouse Mathématiques, Sér 6, Vol 23(4), 2014, 
 p755-778)</p>		
<p>Rabier C-E : <a href="http://projecteuclid.org/euclid.ejs/1414588189"> On the asymptotic robustness of the Likelihood Ratio Test in Quantitative Trait Locus detection </a> (Electronic Journal of Statistics, Vol 8(2), 2014, p2138-2157)
</p>
<p>Rabier C-E : <a href="http://www.tandfonline.com/doi/abs/10.1080/02331888.2013.858720#.UvqQ9fl5OSo"> On stochastic processes for Quantitative Trait Locus mapping under selective genotyping </a> (Statistics, Vol 49(1), 2015, p19-34)
<p>Rabier C-E, Barre P, Asp T, Charmet G, Mangin B : <a href="http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0156086"> On the accuracy of genomic selection </a>  (PLoS One, Vol 11(6), 2016) 
<p style="color:#FF0000";>Data in the CROPDL project : raygrass (INRA Lusignan + Denmark, included in Plos One), wheat (SupAgro Montpellier, INRA Clermont Ferrand). Raygrass data  
 available <a href="https://datadryad.org/resource/doi:10.5061/dryad.jb17n"> here </a> on dryad.
</p>
<p>Rabier C-E, Azais J-M, Elsen J-M, Delmas C : <a href="http://link.springer.com/article/10.1007/s00362-016-0835-y">
Chi-square processes for gene mapping in a population with family structure </a> (Statistical Papers, 2016, doi:10.1007/s00362-016-0835-y) </p> 
<p style="color:#FF0000";>Data in Statistical Papers : human data from  <a href="https://www.genome.gov/10001688/international-hapmap-project/"> HAPMAP project </a>
</p>            
<p>Rabier C-E, Mangin B, Grusea S : <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/sjos.12352"> On the accuracy in high dimensional models and its application to genomic selection</a>, (Scandinavian Journal of Statistics, 2018, to appear), hal-01456310 
<a href="https://hal.archives-ouvertes.fr/hal-01456310"> Version 2 </a>     <a href="https://hal.archives-ouvertes.fr/hal-01456310v1"> Version 1 </a> </p>
<p style="color:#FF0000";>Data in SJS : Rice data from Spindel et al. (<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004982"> Plos Genetics, 2015 </a>), available 
<a href="https://datadryad.org/resource/doi:10.5061/dryad.7369p"> 
here </a> on dryad </p>   
<p> Mangin B, Rincent R, Rabier C-E, Moreau L, Goudemand-Dugue E. <a href="http://charles-elie.rabier.pagesperso-orange.fr/download/PaperEthacOptim.pdf">
Training set optimization
of genomic prediction with EthAcc outperforms usual accuracy and shows that bigger is
not better </a> (Plos One, 2019, to appear).
</p>
<p style="color:#FF0000";>Data in the preprint : Sunflower (see  <a href="http://www.sunrise-project.fr/"> Sunrise Project </a>), Wheat (see <a href="https://breedwheat.fr/"> BreedWheat project </a>), Sugar Beet (see <a href="http://www.aker-betterave.fr/en">AKER project </a>), 
Maize (see  <a href="https://amaizing.fr/"> Amaizing Project </a>) 
</p>




<!-- 
<br><h1>Publications under revision</h1>
<p>Rabier C-E, Azais J-M, Elsen J-M, Delmas C : Chi-square processes for gene mapping in a population with family structure, under Revision for Statistical Papers</p>             
-->

<br><h1>Submitted papers/ in preparation</h1>
<p>Rabier C-E, Delmas C : <a href="https://hal.archives-ouvertes.fr/hal-02123295">
 The SgLasso and its cousins for selective genotyping and extreme sampling: application to association studies and genomic selection
</a>, submitted, hal-02123295 </p>
<p> Previous version was named
<a href="https://hal.archives-ouvertes.fr/hal-01273783"> On gene mapping with the mixture model and the extremes</a>, hal-01273783</p>

<p style="color:#FF0000";>Data in the preprint : Rice data from Begum et al. (<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0119873"> Plos One, 2016 </a>), available <a href="http://www.ricediversity.org/data"> here </a></p>
<p>Recovering phylogenetic network from the study of the mosaic structure of plant genomes,
in preparation with V. Berry, F. Pardi, C. Scornavacca, in preparation</p>
<p style="color:#FF0000";> Data : Rice data from Joao Santos and JC Glazmann (CIRAD), (raw data from the international rice research institute) </p>  
<p> Rabier C-E, Grusea S : <a href="https://hal.archives-ouvertes.fr/hal-01987222">  The accuracy in high dimensional models under imperfect 
linkage disequilibrium </a>, submitted, hal-01987222
</p>
<p style="color:#FF0000";>Data in the preprint: Rice data from Spindel et al. (<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004982"> Plos Genetics, 2015 </a>), available 
<a href="https://datadryad.org/resource/doi:10.5061/dryad.7369p"> 
here </a> on dryad </p>   

<p> Rabier C-E:   The adaptive SgLasso , in preparation
</p>



<h1>PhD Dissertation</h1>
<p>Rabier C-E (2010): Techniques statistiques pour la detection de genes a effets quantitatifs</p>

<h1>Softwares</h1><p> 
<!-- 
<a href="../download/supchi2ornstein.m">supchi2ornstein.m</a> : this function calculates the quantile of the sup of an Ornstein Uhlenbeck chi square 
process on the interval [0;L]. It uses Autoregressive process.<br>
<a href="../download/chi2ornstein.m">chi2ornstein.m</a>: this function simulates one trajectory of an Ornstein Uhlenbeck chi square process on 
the interval [0,L].<br>
<a href="../download/simornstein.m">simornstein.m</a>: this function simulates one trajectory of an Ornstein Uhlenbeck process on 
the interval [0,L].<br>
<a href="../download/supchi2ornsteindelong.m">supchi2ornsteindelong.m</a> : this function calculates the quantile of the sup of an Ornstein Uhlenbeck
process
on the interval [0;L]. It uses Delong's results.<br>
<a href="../download/daviesbound.txt">daviesbound.txt</a> : this function calculates the upper bound for the critical value
for a test at the level alpha. It uses Davies' bound for a chisquare process. The process studied here is a chi square process
with d degrees of freedom on [L,U] and a special covariance.<br>-->

<p>
<a href="../download/PROGRAMSFORSIMULATEDDATA.tar.gz">Genomic Selection</a> : our R code for the Plos One paper ``On the accuracy of genomic selection". It requires the 
R package hypred of Franck Technow. <br>
<br><center><img src="figureGenomicSelection/GraphEmpAcc.PNG" width="300" height="300">
</center>
<br>
<br>
<br>
<p>
<a href="../doc/SPIMAPWGD.html">Phylogenetics</a> : softwares for reconstructing gene trees and detecting Whole Genome Duplications<br>
<br><center><img src="../img/yeastGeneTree.jpg" width="350" height="250">
</center>
<br>
<br>
<br><p>
<a href="../download/imapping.zip">imapping</a> : this is a package with a graphical interface implementing the method
presented in the article "Likelihood Ratio Test Process for Quantitative 
Trait Locus Detection". Download the package and just write pop in your matlab console.<p>
<br>

<center>
<object classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" codebase="http://fpdownload.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=7,0,0,0" width="350" height="360" id="player1" align="middle"> 
<param name="movie" value="imapping_demo.swf"/> 
<param name="menu" value="true"/> 
<param name="quality" value="high"/> 
<param name="bgcolor" value="#FFFFFF"/> 
<noscript><a href="http://www.dvdvideosoft.com">freeware</a></noscript> 
<embed src="imapping_demo.swf" menu="true" quality="high" bgcolor="#FFFFFF" width="350" height="360" name="player" align="middle" allowScriptAccess="sameDomain" type="application/x-shockwave-flash" pluginspage="http://www.macromedia.com/go/getflashplayer"/> 
</object> 
</center>
<br>
<br>
<br>
<br><p>
<a href="../download/imappingfamily.zip">imappingfamily</a> : this is a package with a graphical interface regrouping the different methods
presented in the article "Chi-square processes for gene mapping in a population with family structure". Download the package and just write pop in your matlab console.<p>
<br>

<center>
<object classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" codebase="http://fpdownload.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=7,0,0,0" width="350" height="360" id="player1" align="middle"> 
<param name="movie" value="demoimappingfamily.swf"/> 
<param name="menu" value="true"/> 
<param name="quality" value="high"/> 
<param name="bgcolor" value="#FFFFFF"/> 
<noscript><a href="http://www.dvdvideosoft.com/products/dvd/Free-YouTube-Download.htm">youtube download</a></noscript> 
<embed src="demoimappingfamily.swf" menu="false" quality="high" bgcolor="#FFFFFF" width="350" height="360" name="player" align="middle" allowScriptAccess="sameDomain" type="application/x-shockwave-flash" pluginspage="http://www.macromedia.com/go/getflashplayer"/> 
</object> 
</center>
<br>
<h1>Invited conferences (underlined speaker)</h1><p> 

<p>
<u>C Ané</u>, T Ta, C-E Rabier : Probabilistic approaches for detecting and locating whole genome duplications,
Statistical Methods for Post-Genomic Data, Paris, France, 2014.
</p>
<p><u>C-E Rabier </u> : Gaussian and Chi-Square processes for Quantitative Trait Locus mapping under selective genotyping, International Indian
Statistical Association Conference, Riverside, California, USA, July 2014
</p>
<p><u>B Mangin</u>, C-E Rabier : Training set optimization in genomic selection, 
17th meeting of the EUCARPIA Section Biometrics in Plant Breeding, Ghent, Belgium, 2018.
</p>

<h1>Conferences/Workshops (underlined speaker)</h1><p> 
<p><u>R. Sabbadin</u>, D.Spring, C-E Rabier : Dynamic reserve site selection under contagion risk of deforestation, International Congress on Modelling and Simulation, Melbourne, Australia, 2005.
</p>
<p><u>C-E Rabier</u>, J-M Elsen, C. Delmas : Rejection thresholds in Quantitative Trait Loci detection, 11th QTL-MAS Workshop, Toulouse, 2007
</p>
<p><u>C-E Rabier</u>, J-M Azais : Selective Genotyping for Quantitative Trait Locus detection, Meeting of the MAFIA team of Laboratoire de Statistiques et Probabilites de Toulouse, Nissan lez Enserunes, France, 2007   
</p>
<p><u>C-E Rabier</u>, J-M Azais : Selective Genotyping for Quantitative Trait Locus detection, Statistical Meeting, Santander-Toulouse-Valladolid, Valladolid, Spain, 2008
</p>
<p><u>C-E Rabier</u>, J-M Elsen, C. Delmas : On the theory of Quantitative Trait Locus detection, 24th International Biometric Conference, Dublin, Ireland, 2008

</p>
<p> C-E Rabier, <u>J-M Elsen</u>, C. Delmas : Rejection thresholds in Quantitative Trait Locus detection, Meeting of the European Association for Animal Production, Vilnius, Poland, 2008

<p> <u>C-E Rabier</u>, J-M Azais : Selective Genotyping for Quantitative Trait Locus detection, Meeting Statistics and its applications, Frejus, France, 2008
</p>

<p> <u>C-E Rabier</u>, J-M Azais : Selective Genotyping for Quantitative Trait Locus detection,  Meeting of the French Statistical society, Bordeaux, France, 2009
</p>
<p>
<u>C-E Rabier</u>, J-M Azais, C. Delmas : Likelihood Ratio Test process for the detection of Quantitative Trait Locus, Meeting of the French Statistical society, Marseille, France, 2010
</p>
<p>
<u>C-E Rabier</u> : My research in QTL detection, Stat/Math Phylogenetics group, University of Wisconsin-Madison, USA, 2011
</p>
<p>
<u>C-E Rabier</u>, Delmas C : Likelihood Ratio Test process for Quantitative Trait Loci detection, Joint Statistical Meetings, Miami Beach, USA, 2011
</p>
<p>
<u>C-E Rabier</u>, Ane C : Testing Whole Genome Duplications, Stat/Math Phylogenetics group, University of Wisconsin-Madison, USA, 2012
</p>
<p>
<u>C-E Rabier</u>, Ta T, Ane C : Testing Whole Genome Duplications, Evolution Seminar Series, University of Wisconsin-Madison, USA, 2013
</p>
<p>
<u>C-E Rabier</u>, Ta T, Ane C : Testing Whole Genome Duplications, Evolution 2013, Snowbird, USA.</p>
<p>
<u>C Ané</u>, Ta T, Rabier C-E : Probabilistic approaches for detecting and locating whole genome duplications,
Seminar of the laboratory of Biometry and Evolutive Biology, University Lyon 1, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics Seminar, Mathematics Research Institute of Rennes, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics and Probability Seminar, University of Nice, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics Seminar, University Joseph Fourier, Grenoble, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian processes and Phylogenetic trees for the genome. Statistics Seminar, Statistics and Genome, University of Evry, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics and Probability Seminar, University of Angers, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian processes and Phylogenetic trees for the genome. Mathematics, Evolution and Genomic Seminar, University Aix-Marseille, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics and Probability Seminar, University of Pau, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Statistics Seminar, University Lyon 1, France, 2014.
</p>
<p>
<u>C-E Rabier</u>: Ta T, Ane C. Testing Whole Genome Duplications. MIAT Unit Seminar, INRA Toulouse, France, 2014.</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Seminar of MODAL team, INRIA Lille, 2015.
</p>
<p>
<u>C-E Rabier</u>: Ta T, Ane C. Testing Whole Genome Duplications. Seminar of GenPhySE Bionformatics team, INRA Toulouse, France, 2015. 
<a href="../download/talkGenPhysewithoutheader.pdf"><button class="button button2">Slides</button></a>
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Seminar of Mathematics, University of South Brittany, France, 2015.
</p>
<p>
<u>C-E Rabier</u>: Gaussian and Chi-Square processes for Quantitative Trait Locus detection. Seminar of Probability and Statistics, University of Bordeaux 1, France, 2015.
<a href="../download/talkUnivBordeaux.pdf"><button class="button button2">Slides</button></a>
</p>
<p>
<u>C-E Rabier</u>, Ta T, Ane C: Testing Whole Genome Duplications. Seminar of the Methods and Algorithms for Bioinformatics (MAB) group, LIRMM, Montpellier, 2015. 
</p>
<p>
<u>C-E Rabier</u>:  Gaussian processes and phylogenetic trees in genomics. Statistics seminar, ENSAI, Rennes, France, 2015.
<a href="../download/talkENSAI.pdf"><button class="button button2">Slides</button></a>
</p>
<p>
<u>
C-E Rabier </u>, Barre P, Charmet G, Mangin B : On the accuracy of genomic selection. IMS-China International Conference on Statistics and Probability, Kunming, China 2015.
</p>
<p>
<u>
C-E Rabier </u>, Barre P, Charmet G, Mangin B : On the accuracy of genomic selection.  MIAT Unit Seminar, INRA Toulouse, France, 2015.
</p>
<p>
<u>C-E Rabier</u>: Gaussian processes for gene mapping. Statistics seminar, University of Strasbourg, France, 2015.
</p>
<p>
C-E Rabier, <u>C Delmas</u>: On gene mapping with the mixture model and the extremes. Meeting of the French Statistical society, Montpellier, France, 2016.
</p>
<p>
<u>C-E Rabier</u>: My research in statistical genetics. Working group Genome Harvest,
CIRAD, Montpellier, France, 2016.
</p>
<p>
<u> C-E Rabier</u>, V. Berry, F. Pardi, C. Scornavacca: Modèles et outils à l'interface entre la
génétique des populations et de la
phylogénétique pour la phylogénomique du riz. Working group Genome Harvest,
CIRAD, Montpellier, France, 2017.
</p>
<p>
<u> C-E Rabier</u>, V. Berry, F. Pardi, C. Scornavacca: Apport des approches phylogénétiques pour
expliquer l'origine des génomes mosaïques,
exemple chez le Riz, Genome Harvest day, 
CIRAD, Montpellier, France, 2017.
<button class="button button2" onClick="passWordGHDay()">Slides</button>
</p>
<p>
<u> C-E Rabier</u>, B.Mangin: On prediction in genomic selection
, R2D2 Meeting (working group from INRA), 
Noirmoutiers, France, 2017.
<button class="button button2" onClick="passWordR2D2()">Slides</button>
</p>
<p>
<u> C-E Rabier</u>: Apprentissage en selection génomique, et reconstruction d'arbres et de réseaux en phylogénie, 
séminaire de l'équipe Méthodes et Algorithmes pour la Bioinformatique, LIRMM Montpellier, France, 2018.
<button class="button button2" onClick="passWordMAB()">Slides</button>
</p>
<p>
<u> C-E Rabier</u>, V. Berry, F. Pardi, C. Scornavacca: Apport des approches phylogénétiques pour
expliquer l'origine des génomes mosaïques,
 exemple chez le Riz. 9èmes Journées Francophones sur les Réseaux Bayésiens et les Modèles Graphiques Probabilistes, Toulouse, France, 2018.
<button class="button button2" onClick="passWordJFRB()">Slides</button>
</p>
<p>
<u> C-E Rabier</u>, V. Berry, M. Labous, F. Pardi, C. Scornavacca: Apport des approches phylogénétiques pour
expliquer l'origine des génomes mosaïques,
 exemple chez le Riz. Working group Genome Harvest, CIRAD, Montpellier, France, 2018.
<button class="button button2" onClick="passWordGHLastMe()">Slides Method SimulData Rabier</button>
<button class="button button2" onClick="passWordGHLastVincent()">Slides RealData VBerry</button>
</p>
<p>
<u> C-E Rabier</u> : Prédiction en grande dimension, extrêmes et réseaux. Séminaire de l'ISAE-Supaero, Toulouse, France, 2018.
<button class="button button2" onClick="passWordISAE()">Slides</button>
</p>




</div>

<div class="webdesign"><p><a href="http://www.sarkis-webdesign.com/" title="Webdesign" class="web_design">Webdesign</a> service by Sarkis. <a
href="http://freelancewebmarket.com/" title="Outsourcing" class="outsourcing">Outsourcing</a> by FreelanceWebmarket.</p></div>

 <div class="footer">&copy; 2007 <a href="index.html">Website.com</a>. Valid <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> &amp; <a
href="http://validator.w3.org/check?uri=referer">XHTML</a>. <a href="http://templates.arcsin.se">Website template</a> by <a
href="http://arcsin.se">Arcsin</a>

 </div>

</div>

</body>

</html>
